src/styles/index.scss

```diff
@@ -1,3 +1,120 @@
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.card {
+  width: 200px;
+  padding: 16px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background-color: white;
+  text-align: center;
+  margin: 0 auto;
+}
+
+.imac img {
+  margin-top: 16px;
+  width: 160px;
+  height: 134px;
+}
+
+.card__title {
+  color: #060b35;
```

Comment:
this color is used in multiple places. Its good to keep it in variable.
Also remember about checklist
-  SCSS Variables are used for main values and placed in a separate file